src/CreatorToken.sol

```diff
@@ -124,10 +176,20 @@ contract CreatorToken is ERC721Royalty {
     payToken.safeTransferFrom(msg.sender, admin, _adminFee);
   }
 
+  /// @notice Buy multiple creator tokens in bulk and send them to the caller.
+  /// @dev The recipient of the creator tokens is the msg.sender.
+  /// @param _numOfTokens Number of tokens to buy.
+  /// @param _maxPayment The maximum amount of USDC the caller is willing to pay.
+  /// @return _totalPrice The total amount of USDC paid for the creator tokens.
   function bulkBuy(uint256 _numOfTokens, uint256 _maxPayment) public returns (uint256 _totalPrice) {
     _totalPrice = bulkBuy(msg.sender, _numOfTokens, _maxPayment);
   }
 
+  /// @notice Buy multiple tokens in bulk and send them to a specified address.
+  /// @param _to Address where the tokens should be sent.
+  /// @param _numOfTokens Number of tokens to buy.
+  /// @param _maxPayment The maximum amount of USDC the caller is willing to pay.
+  /// @return _totalPrice  The total amount of USDC paid for the creator tokens.
```

Comment:
nit: extra space

Reply:
removed in https://github.com/showtime-xyz/creator-tokens/commit/3c46721e1c9a0899f54630e2b73028cd5bef159c